composer.json

```diff
@@ -35,6 +34,12 @@
         "symfony/yaml": "^6.2.5",
         "vimeo/psalm": "^5.6.0"
     },
+    "suggest": {
+        "doctrine/cache": "Library deprecated, nevertheless Doctrine\\Common\\Cache\\Cache implementations wrapped into the bundled PSR-6 adapter"
+    },
+    "conflict": {
+        "doctrine/cache": "<2.0"
+    },
```

Comment:
I don't think we need to conflict with this.

Reply:
ok, will remove to allow clients implement factories and continue use v1